recipes/gangstr/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" ..
+make install
+# TODO: copy python scripts
```

Comment:
Those are some Python 2 scripts... Are they important? Do we need/want to include them?

Reply:
no, this is a relict, we do not need them

Reply:
Perfect :)